tests/codeception/acceptance/Install/Content/FlexiblePageCest.php

```diff
@@ -7,100 +7,6 @@
  */
 class FlexiblePageCest {
 
-  protected $disableCollection = FALSE;
```

Comment:
The following tests were moved to `tests/codeception/functional/Install/Content/FlexiblePageCest.php`: 
testPostCard
testAccordion
testBackToTopExists
testTextArea
testCollections

This was done due to the increased load time for the paragraphs module and for adding components. The `FunctionalTester` has the wait functions necessary to run the tests.